packages/x-date-pickers/src/internals/models/common.ts

```diff
@@ -2,6 +2,8 @@ import { DateView, TimeView } from '../../models/views';
 
 export type WrapperVariant = 'mobile' | 'desktop' | null;
 
+export type ValueType = 'date' | 'time' | 'date-time' | null;
```

Comment:
We already have the `FieldValueType` type. 🤔

Reply:
It makes sense to have a single source of truth and move the type "higher up". 👌 👍